tests/DocumentationExamplesTest.php

```diff
@@ -31,24 +31,9 @@
  */
 class DocumentationExamplesTest extends FunctionalTestCase
 {
-    public function setUp(): void
-    {
-        parent::setUp();
-
-        $this->dropCollection();
-    }
-
-    public function tearDown(): void
-    {
-        if (! $this->hasFailed()) {
-            $this->dropCollection();
-        }
-
-        parent::tearDown();
-    }
-
     public function testExample_1_2(): void
     {
+        $this->dropCollection(null, 'inventory');
```

Comment:
It is more verbose to explicitly define the name of the collection in each test case, but this is how I can handle the variety of names used in the documentation.